react-nginx/docker-compose.yml

```
@@ -0,0 +1,8 @@
+version: "2"
+services:
+  frontend:
+    build:
+      context: .
+    container_name: frontend
+    ports:
+      - "3000:80"
```

For the samples, we try to keep the default port of the tool used, here you're using Nginx so we should expose 80 to the 80 port of the host
```suggestion
      - "80:80"
```

---

It's not resolved for me, because your using Nginix to expose the application, we should keep default port this tool, so we should bind `80:80` and not `3000:80`